platform.md

```diff
@@ -839,7 +826,18 @@ Usage:
     - The value of `io.buildpacks.lifecycle.metadata` SHALL be modified as follows
       - `run-image.reference` SHALL uniquely identify `<run-image>`
       - `run-image.top-layer` SHALL be set to the uncompressed digest of the top layer in `<run-image>`
-    - The value of `io.buildpacks.stack.*` labels SHALL be modified to that of the new `run-image`
+    - The value of `io.buildpacks.base.*` labels and `io.buildpacks.stack.*` labels (if present) SHALL be modified to that of the new `run-image`
+- **If** `<force>` is `true`, the following [target data](#target-data) values in the output `<image>` config MUST be derived from the new `<run-image>`:
+  - `os`
+  - `architecture`
+  - `variant` (if specified)
+  - `io.buildpacks.distribution.name` (if specified)
+  - `io.buildpacks.distribution.version` (if specified)
+- **Else** they MUST match the old run image if `<force>` is `false`
+- **If** `<force>` is `true` and the provided `<run-image>` is not found in `runImage.image` or `runImage.mirrors`:
+  - `run-image.image` SHALL be the provided `<run-image>`
+  - `run-image.mirrors` SHALL be omitted
+- **Else** the provided `<run-image>` MUST be found in `runImage.image` or `runImage.mirrors` if `<force>` is `false`
```

Comment:
Does it make sense to make this an `**Else If**`?

```suggestion
- **Else If** `<force> is `false`, the provided `<run-image>` MUST be found in `runImage.image` or `runImage.mirrors`
```

Reply:
Updated capitalization, but yes this reads better 🙏🏼